projects/packages/jetpack-mu-wpcom/src/features/wpcom-widgets/wpcom-widgets.php

```diff
@@ -0,0 +1,101 @@
+<?php
```

Comment:
Most of this file is copied from fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sjcpbz%2Qjvqtrgf.cuc-og as a way of managing the widgets. Eventually that file on WPcom could get deleted.